docs_user/modules/con_fips-support.adoc

```diff
@@ -0,0 +1,12 @@
+[id="fips-support_{context}"]
+
+= FIPS support
+
+
+There is a major difference with how FIPS was configured in {OpenStackPreviousInstaller} and how it is enabled and enforced in operator deployments.
+
+In {OpenStackPreviousInstaller} FIPS is enabled as part of it's configuration.
+
+In operator deployments there is no specific FIPS configuration at the operator level. If your deployment has FIPS compliance requirements, the whole {OpenShift} ({OpenShiftShort}) cluster must be installed with FIPS being enabled. This will have happened before proceeding with this guide and installing {OpenStackShort}.
+
+Please refer to the link:https://docs.openshift.com/container-platform/latest/installing/installing-fips.html[Support for FIPS cryptography] in _OpenShift Container Platform Documentation_.
```

Comment:
```suggestion

There is a difference between how Federal Information Processing Standard (FIPS) was configured in {OpenStackPreviousInstaller} and how it is enabled and enforced in operator deployments.

* In {OpenStackPreviousInstaller}, FIPS is enabled as part of its configuration.
* In operator deployments, there is no specific FIPS configuration at the operator level. If your deployment has FIPS compliance requirements, you must install the whole {OpenShift} ({OpenShiftShort}) cluster with FIPS enabled. The installation happens before you proceed with the adoption and install {rhos_prev_long}.

For more information about FIPS support, see link:https://docs.openshift.com/container-platform/latest/installing/installing-fips.html[Support for FIPS cryptography] in _OpenShift Container Platform Documentation_.
```

Reply:
When you say, "FIPS is enabled as part of its configuration". do you mean that FIPS is enabled by default? 
I yes, I think it would be clearer to say "In director, FIPS is enabled by default."

Reply:
No, FIPS was not enabled by default.  Rather, there was an additional template that enabled certain THT parameters that were fips specific.

Reply:
I adjusted the wording, it should be more clear now.